plugins/typescript/src/templates/context.ts

```diff
@@ -22,10 +22,6 @@ export const getContext = (prefix: string, componentsFile: string) =>
        */
       enabled?: boolean;
```

Comment:
We need to adjust the type of the `enabled` param, after some manual integration test with the github example I got this type error:

![image](https://github.com/user-attachments/assets/f2bc93d0-a130-422b-85cd-cd92302d8908)

Note to myself: some love is needed for the integration test, it’s a bit painful to setup right now